projects/ngx-text-diff/src/lib/ngx-text-diff.component.ts

```diff
@@ -14,13 +14,14 @@ export class NgxTextDiffComponent implements OnInit {
   @Input() right = '';
   @Input() loading = false;
   @Input() diffContentObservable$: Observable<DiffContent>;
+  @Input() showBtnToolbar = true;
   subscriptions: Subscription[] = [];
```

Comment:
I see we are using this subscriptions array - but we are never unsubscribing once the component is destroyed. I would recommend to add `OnDestroy()` and unsubscribe from all subscriptions.